src/components/CurriculumComponents/CurriculumDownloadTab/helper.test.ts

```diff
@@ -0,0 +1,110 @@
+import { renderHook } from "@testing-library/react";
+
+import {
+  defaultValueWhenThrown,
+  saveDownloadsDataToLocalStorage,
+  useDownloadsLocalStorage,
+} from "./helper";
+
+import {
+  LS_KEY_EMAIL,
+  LS_KEY_SCHOOL,
+  LS_KEY_TERMS,
+} from "@/config/localStorageKeys";
+
+describe("CurriculumDownloadTab / helper", () => {
+  beforeEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  describe("defaultValueWhenThrown", () => {
```

Comment:
Is this test checking for an error to be thrown or the error message?

Reply:
This helper just returns a default value if the thing running inside the function throws.